app/state/selectors/authSelectors.js

```diff
@@ -23,6 +23,13 @@ function isLoggedInSelector(state) {
   return Boolean(state.auth.userId && state.auth.token);
 }
 
+/**
+ * TODO: Find out if this is needed any more, and if isn't: Remove.
+ * With our current knowledge, people are given admin permissions for
+ * resources through: https://respa.koe.hel.ninja/admin/resources/unitauthorization
```

Comment:
I dont think we should expose this, cause varaamo is kinda open-source. This link belong to admin, which should not be opened?